wfe/web-front-end.go

```diff
@@ -708,7 +708,11 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 
 		cert, err := wfe.SA.GetCertificateByShortSerial(serial)
 		if err != nil {
-			wfe.sendError(response, "Not found", err, http.StatusNotFound)
+			if strings.HasPrefix(err.Error(), "gorp: multiple rows returned") {
```

Comment:
I was thinking of Select instead of SelectOne:

> Values are returned in one of two ways: 1. If i is a struct or a
pointer to a struct, returns a slice of pointers to matching rows of
type i. 2. If i is a pointer to a slice, the results will be appended to
that slice and nil returned.